lib/transforms/subsetFonts.js

```diff
@@ -249,7 +306,11 @@ async function getSubsetsForFontUsage(assetGraph, htmlAssetTextsWithProps, forma
 
         for (const item of htmlAssetTextsWithProps) {
             for (const fontUsage of item.fontUsages) {
-                if (!fontUsage.props.src.includes('//fonts.gstatic.com')) {
+                if (!fontUsage.fontUrl) {
+                    continue;
+                }
+
+                if (!fontUsage.fontUrl.includes('//fonts.gstatic.com')) {
```

Comment:
Not watertight. Maybe you could pick out `asset.hostname`?

Reply:
Done